perfscale_regression_ci/scripts/scalability/node-affinity-anti-affinity.sh

```diff
@@ -0,0 +1,162 @@
+#/!/bin/bash
+################################################
+## Auth=prubenda@redhat.com lhorsley@redhat.com
+## Desription: This testcase tests Node Affinity and anti-affinity as we approach to node capacity	
+## Polarion test case: OCP-18082 - NextGen Pod scheduler at capacity with Node Affinity and Anti-Affinity rules	
+## https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-18082
+## Cluster config: Cluster needed in AWS EC2 (m5.xlarge, 3 master/etcd, 3 worker/compute nodes)
+## kube-burner config: perfscale_regerssion_ci/kubeburner-object-templates/node-affinity-anti-affinity-config.yml
+################################################ 
+
+
+source ../../utils/run_workload.sh
+source ../custom_workload_env.sh
+source ../common.sh
+source node-affinity-anti-affinity_env.sh
+
+function show_node_labels() {
+  oc get node --show-labels
+  oc get node -l cpu=4
+  oc get node -l cpu=6
+  oc get node -l beta.kubernetes.io/arch=intel
+}
+
+
+export ANTI_AFFINITY_JOB_ITERATION=${ANTI_AFFINITY_JOB_ITERATION:-190}
+export AFFINITY_JOB_ITERATION=${AFFINITY_JOB_ITERATION:-190}
+
+# Output some general information about the test environment
+date
+uname -a
+oc get clusterversion
+oc version
+oc get node --show-labels
+oc describe node | grep Runtime
+
+
+echo "======Setup/Configuration: label nodes for Affinity and anti-affinity scheduling======"
+
+compute_nodes=$(oc get nodes -l 'node-role.kubernetes.io/worker=' | awk '{print $1}' | grep -v NAME | xargs)
+
+echo -e "\nWorker  nodes are: $compute_nodes"
+
+declare -a node_array
+counter=1
+
+oc get nodes -l 'node-role.kubernetes.io/worker='
+oc describe nodes -l 'node-role.kubernetes.io/worker='
```

Comment:
let's maybe move this to after the run workload is done. A check point of this test is to see how much cpu and memory we can push on a node, so this describe would be better seen after the wokrload is run before delete